src/hs111.jl

```diff
@@ -0,0 +1,50 @@
+# Hock and Schittkowski problem number 111.
+#
+#   Source:
+#   Problem 111 in
+#   W. Hock and K. Schittkowski,
+#   Test examples for nonlinear programming codes,
+#   Lectures Notes in Economics and Mathematical Systems 187,
+#   Springer Verlag, Heidelberg, 1981.
+#
+#   classification QQR2-AN-2-1
+#
+# N. Antunes, Curitiba, 10/2016.
+
+export hs111
+
+"HS111 model"
+function hs111(args...)
+
+  nlp = Model()
+
+  c = [-6.089, -17.164, -34.054, -5.914, -24.721, -14.986, -24.100, -10.708, -26.662, -22.179]
+
+  @variable(nlp, x[i=1:10], lowerbound = -100, start = -2.3, upperbound = 100)
```

Comment:
In this case, woulnd't it be more readable to write `@variable(nlp, -100 ≤ x[i=1:10] ≤ 100, start=-2.3)`? I think JuMP allows such compact syntax (or something close to it).